test/unit/nedap/utils/test/api.cljc

```diff
@@ -168,13 +176,20 @@
                                        form
                                        @test-result))
           (sut/expect 0 :to-change 0 :from 0 :to 1)
-          `{:type :fail, :expected (impl/meta= [0 1]), :actual (~'not (impl/meta= [0 1]))}
+          `{:type :fail, :expected (sut/meta= 0 1), :actual (~'not (sut/meta= 0 1))}
 
           (sut/expect 0 :to-change {} :from {} :to ^::test {})
-          `{:type :fail, :expected (impl/meta= [{} {}]), :actual (~'not (impl/meta= [{} {}]))}
+          `{:type :fail, :expected (sut/meta= {} {}), :actual (~'not (sut/meta= {} {}))}
 
           (sut/expect (swap! a inc) :to-change @a :from 0 :to 2)
-          `{:type :fail, :expected (impl/meta= [(deref ~'a) 2]), :actual (~'not (impl/meta= [1 2]))}))))
+          `{:type :fail, :expected (sut/meta= (~'clojure.core/deref ~'a) 2), :actual (~'not (sut/meta= 1 2))}
+
+           ;; change matcher to `=`
+          (sut/expect 0 :to-change 0 :from 0 :to 1 :with =)
+          `{:type :fail, :expected (~'= 0 1), :actual ~'(not (= 0 1))}
+
+          (sut/expect (swap! a inc) :to-change @a :from 1 :to 3 :with =)
+          `{:type :fail, :expected (~'= (~'clojure.core/deref ~'a) 3), :actual ~'(not (= 2 3))}))))
```

Comment:
Check out https://app.circleci.com/pipelines/github/nedap/utils.test/159/workflows/12ed82f3-565c-40a8-9c55-384d8102eb15/jobs/318 , a warning is emitted

Reply:
do you mean
> WARNING: Use of undeclared Var matcher-combinators.matchers/Absent at line 169 target/out/matcher_combinators/matchers.cljc


I checked the [source](https://github.com/nubank/matcher-combinators/blob/9e3a7899c09b5446fc2c884e74092589fe1f009e/src/cljc/matcher_combinators/matchers.cljc#L169), and we don't use the `match-with` matcher right now in the tests. Made an issue here: https://github.com/nubank/matcher-combinators/issues/154